spec/features/invoices/index_spec.rb

```diff
@@ -35,4 +35,42 @@
       expect(subject).to_not have_content(org2_invoice.customer)
     end
   end
+
+  context 'complete invoice', js: true do
+    let!(:account)  { create :bank_account, organization: org }
+    let!(:category) { create :category, :income, organization: org }
+    let!(:invoice)  { create :invoice, organization: org, amount: 500 }
+    let(:comission) { Money.new(100, invoice.currency) }
+
+    before do
+      visit invoices_path
+      click_on 'Complete Invoice'
+      within '#new_transaction' do
+        select category.name, from: 'transaction[category_id]'
+        select account.name, from: 'transaction[bank_account_id]'
+        fill_in 'transaction[comission]', with: comission
+        fill_in 'transaction[comment]', with: 'TestComment'
+      end
+      click_on 'Create'
+      visit invoices_path
```

Comment:
@yunixon move `visit invoices_path` for separated case. You should only Create Transaction in this before block.